docker-entrypoint.sh

```diff
@@ -3,6 +3,10 @@ set -e
 
 if [ "$1" = 'gunicorn' ]; then
     ./manage.py migrate --noinput
+    until curl -sf http://localhost:9200 > /dev/null; do 
```

Comment:
What if Elasticsearch fails to startup?

Reply:
Good point. Let’s define a readiness probe for the Elasticsearch container in Kubernetes, so that the Dienst2 pod will fail to start when Elasticsearch fails to start.

Reply:
👍

Reply:
Done